dags/openshift_nightlies/config/install/aws/sdn.json

```diff
@@ -10,24 +10,24 @@
     "openshift_network_type": "OpenShiftSDN",
     "openshift_master_instance_type": "r5.4xlarge",
     "openshift_worker_instance_type": "m5.2xlarge",
-    "openshift_master_root_volume_size": 220,
+    "openshift_master_root_volume_size": 100,
     "openshift_master_root_volume_type": "gp3",
     "openshift_master_root_volume_iops": 3000,
-    "openshift_worker_root_volume_size": 100,
+    "openshift_worker_root_volume_size": 50,
     "openshift_worker_root_volume_type": "gp3",
     "openshift_worker_root_volume_iops": 0,
     "machineset_metadata_label_prefix": "machine.openshift.io",
     "openshift_infra_node_instance_type": "m5.12xlarge",
```

Comment:
Did we want to reduce the instance size here?

Reply:
I've verified that we're far from using more than 50GiB in our regular 27 nodes tests